content/en/docs/languages/java/instrumentation.md

```diff
@@ -1,1150 +1,220 @@
 ---
-title: Instrumentation
+title: Instrumentation ecosystem
 aliases:
   - /docs/java/getting_started
   - /docs/java/manual_instrumentation
   - manual
   - manual_instrumentation
-weight: 20
-description: Manual instrumentation for OpenTelemetry Java
-# prettier-ignore
-cSpell:ignore: Autowired customizer logback loggable multivalued rolldice springframework
+weight: 10
+description: Instrumentation ecosystem in OpenTelemetry Java
+cSpell:ignore: Logback logback
 ---
 
 <!-- markdownlint-disable no-duplicate-heading -->
 
-{{% docs/languages/instrumentation-intro %}}
+Instrumentation records telemetry using the [API](../api-components/). The
+[SDK](../sdk/) is the built-in reference implementation of the API, and is
+[configured](../configuration/) to process and export the telemetry produced by
+instrumentation API calls. This page discusses the OpenTelemetry ecosystem in
+OpenTelemetry Java, including resources for end users and cross-cutting
+instrumentation topics:
 
-{{% alert title="Note" color="info" %}} See [Manage Telemetry with SDK](../sdk/)
-for a conceptual overview of OpenTelemetry Java SDK concepts. See
-**[Configure the SDK](../configuration/)** for details on SDK configuration,
-including
-[zero-code SDK autoconfigure](../configuration/#zero-code-sdk-autoconfigure).
-{{% /alert %}}
+- [Instrumentation categories](#instrumentation-categories): There are a variety
+  of categories of instrumentation addressing different use cases and
+  installation patterns.
+- [Context propagation](#context-propagation): Context propagation provides
+  correlation between traces, metrics, and logs, allowing the signals to
+  complement each other.
+- [Semantic conventions](#semantic-conventions): The semantic conventions define
+  how to produce telemetry for standard operations.
+- [Log instrumentation](#log-instrumentation): The semantic conventions define
+  how to produce telemetry for standard operations.
 
-{{% alert title="Note" color="info" %}}
+{{% alert %}} While [instrumentation categories](#instrumentation-categories)
+enumerates several options for instrumenting an application, we recommend users
+start with the [Java agent](#zero-code-java-agent). The Java agent has a simple
+installation process, and automatically detects and installs instrumentation
+from a large library. {{% /alert %}}
```

Comment:
@open-telemetry/java-approvers, @open-telemetry/java-instrumentation-approvers per conversation in last week's java SIG, I've added this call out to indicate our recommendation to start with the java agent.